prometheus-exporters/vrops-exporter/templates/vrops-exporter-deployment.yaml

```diff
@@ -29,6 +36,12 @@ spec:
            args:
              - -m
              - /config/collector_config.yaml
+             - -t
+             - {{ required "$target.hostname missing" $target.hostname }}
```

Comment:
is "required" reasonable for the default case?

Reply:
yes, in case it is not defined